SPECS-EXTENDED/memkind/memkind.spec

```diff
@@ -1,26 +1,43 @@
+%global gittag0 v1.14.0
+# WORKAROUND to avoid breaking the build at the atrocious libtool shell scrip
+# due to RPM environmental macros being lost for the subshells
+%undefine _package_note_file
+
+Name: 		memkind
+Summary: 	User Extensible Heap Manager
+Version: 	1.14.0
+Release: 	10%{?dist}
+License: 	BSD-3-Clause
+URL: 		http://memkind.github.io/memkind
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
-%global gittag0 v1.10.0
-
-Name: memkind
-Summary: User Extensible Heap Manager
-Version: 1.10.0
-Release: 2%{?dist}
-License: BSD
-URL: http://memkind.github.io/memkind
-BuildRequires: automake libtool numactl-devel systemd gcc gcc-c++
-
-# x86_64 is the only arch memkind will build and work due to
-# its current dependency on SSE4.2 CRC32 instruction which
-# is used to compute thread local storage arena mappings
-# with polynomial accumulations via GCC's intrinsic _mm_crc32_u64
-# For further info check: 
-# - /lib/gcc/<target>/<version>/include/smmintrin.h
-# - https://gcc.gnu.org/bugzilla/show_bug.cgi?id=36095 
-# - http://en.wikipedia.org/wiki/SSE4
-ExclusiveArch: x86_64
-
-Source0: https://github.com/%{name}/%{name}/archive/%{gittag0}/%{name}-%{version}.tar.gz
+
+BuildRequires: 	make
+BuildRequires:	patch
+BuildRequires: 	automake
+BuildRequires:	libtool
+BuildRequires:	numactl-devel
+BuildRequires:	systemd
+BuildRequires:	gcc
+BuildRequires:	gcc-c++
+BuildRequires:	daxctl-devel
+# memkind has been discontinued and archived upstream. See Bugzilla 2296288.
+# We are deprecating it now, to flag any potential user of its future removal.
+Provides: 	deprecated()
+
+# Upstream testing of memkind is done exclusively on x86_64; other archs
+# are unsupported but may work.
+ExclusiveArch: 	x86_64
+ExclusiveArch:	ppc64
```

Comment:
We dont  support all archs but x86_64 & aarch64, so we can remove ExclusiveArch .